ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__4.9.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  base:
+    name: cucushift
+    namespace: ci
+    tag: cucushift
+build_root:
+  image_stream_tag:
+    name: verification-tests
+    namespace: ci
+    tag: centos
+images:
+- dockerfile_path: tools/openshift-ci/Dockerfile
+  from: base
+  to: verification-tests
+promotion:
+  namespace: ci
+  tag: "4.9"
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.9"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.9"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
```

Comment:
This file configures the container build test upon PR creations, release/ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml is the file to update to configure periodic jobs.

Reply:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml#L472

Reply:
No. This was on purpose.  See explanation in the description.

Reply:
I've tried the way you suggested in https://github.com/openshift/release/pull/21879/files, and I would prefer the way used here, as it does not rely on configs/changes in ci-operator/config/openshift/release/

Reply:
Nice idea!